draft-jones-perc-dtls-tunnel/draft-jones-perc-dtls-tunnel.md

```diff
@@ -317,7 +326,8 @@ than one tunnel has been established between it and a key distributor.
 prefer to allow the media distributor to send messages over more
 than one tunnel to more than one key distributor?  The latter would
 provide for higher availability, but at the cost of key distributor
-complexity.
+complexity. The former would allow the usage of a load distributor in
+front of the key distributor.
```

Comment:
One typical option for simple load balancing is to use a TCP connection based load balancing. But to be able to use such a load balancer all message for a given association ID would have to go down the same TCP connection so they get to the same KD (except if the multiple KDs synchronizes among them somehow, which to some degree defeats the purpose of load balancing). Multiple connections to the KD sounded to me like these would not necessarily all go to the exact same instance of the KD. But maybe I'm just mis-reading this.
Having said all of this: it currently appears doubtful to me why someone would need to scale the KD horizontally, as I assume the KD's job is pretty light weight. So I feel that if at all multiple MD's will use the same KD and therefore the KD needs to send every message over the same connection it received messages for the association ID for on.
Which actually makes me think we should add stronger wording in the section "Key Distributor Tunneling Procedure" to ensure messages get send back over the same TLS connection. It also raises the question if we should say anything about association ID collisions coming from different MD's. I guess we could or should suggest that the association ID in combination with the TLS connection file descriptor or the cert from other side should be used to be able to handle multiple identical association IDs.